core/src/main/resources/error/error-classes.json

```diff
@@ -469,6 +469,11 @@
       "Grouping sets size cannot be greater than <maxSize>"
     ]
   },
+  "GROUP_BY_AGGREGATE" : {
```

Comment:
Could we re-use `GROUP_BY_POS_REFERS_AGG_EXPR ` ?

Reply:
I'm not pretty sure since `GROUP_BY_POS_REFERS_AGG_EXPR` requires <index>, but it's only care if the aggregate function is used in the group by cause.

We can manually pass the missing value for <index>, but not sure it makes sense or not.

WDYT, @srielau @MaxGekk ?

Reply:
I like it that they are distinct. GROUP_BY_POS_... might be fat fingering, GROUP_BY_AGGREGATE is: You don't know how GROUP BY works...

Have reviewed? GROUP_BY_POS_REFERS_AGG_EXPR... that's not very pretty.

Reply:
> Have reviewed? GROUP_BY_POS_REFERS_AGG_EXPR... that's not very pretty.

Yes, this is the merged code, but we can rename it